packages/core/core/src/registries/policies.ts

```diff
@@ -108,7 +109,22 @@ const policiesRegistry = () => {
      * Returns a map with all the policies in a namespace
      */
     getAll(namespace: string) {
-      return pickBy((_, uid) => hasNamespace(uid, namespace))(Object.fromEntries(policies));
+      const filteredPolicies = pickBy((_, uid) => hasNamespace(uid, namespace))(
+        Object.fromEntries(policies)
+      );
+
+      // Return an enumerable object of the same format as controllers, services, etc
```

Comment:
I don't see why this is necessary ? the picKBy should already return a enumerable object 🤔

Reply:
Oops, you're right, that one can be left alone I think.